tests_and_analysis/test/data/phonopy_files/README

```diff
@@ -0,0 +1,7 @@
+The Phonopy force constants and structure data in this directory and
+subdirectories are from the Kyoto Phonon database at
+http://phonondb.mtl.kyoto-u.ac.jp/. The contents of the Kyoto Phonon
+database website are licensed under a Creative Commons Attribution
```

Comment:
Looking at the Creative Commons guidance on how to attribute CC-by, they consider an "ideal" attribution to have:
- Title (Done)
- Creator
- Source (Done)
- License (Done, a link is also fine)

So I suggest we make this "The contents of the Kyoto Phonon database website, by Atsushi Togo, are licensed..."

Reply:
Done!